### PR TITLE
Add cli args description to readme

### PR DESCRIPTION
**Pull request by fmoletta**
Solves #1349

### CODE REVIEW

**Review by Oppen — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -135,7 +135,29 @@ cargo build --release
...
+
+- `--memory_file <MEMORY_FILE>` : Receives the name of a file and outputs the relocated memory into it
+
+- `--print_output` : Prints the program output
+
+- `--proof_mode`: Runs the program in proof_mode
+
+- `--secure_run`: Runs security checks after execution. Enabled by default when not in proof_mode
+
+- `--air_public_input <AIR_PUBLIC_INPUT>`: Receives the name of a file and outputs the AIR public inputs into it. Can only be used if proof_mode is also enabled.
```
Comment (pefontana) [inline README.md:155]: I think that to use the --air_public_input is also required to use the --trace_file and -memory_file tags

File: README.md [hunk 2]
```diff
@@ -135,7 +135,29 @@ cargo build --release
...
+
+For example, to obtain the air public inputs from a fibonacci program run, we can run :
+
+```bash
+  target/release/cairo-vm-cli cairo_programs/fibonacci.json --layout all_cairo --proof_mode --air_public_input fibonacci_public_input.json
```
Comment (pefontana) [inline README.md:160]: Here i will use a program that is compiled with proof-mode in the Makefile

Comment (fmoletta) [reply README.md:160]: oh right

Comment (fmoletta) [reply README.md]: fixed

**Review by pefontana — APPROVED** [review]

**Pull request merged**